src/commands/utils/ping.ts

```diff
@@ -5,6 +5,7 @@ export const ping: CommandDefinition = {
     name: 'ping',
     description: 'Send back a message',
     category: CommandCategory.UTILS,
+    requiredPermissions: ['BAN_MEMBERS'],
```

Comment:
```suggestion
    requiredPermissions: ['MANAGE_WEBHOOKS'],
```

This will make it so only both bot devs and moderators can use it.

Reply:
mod team don't have manage webhooks
![image](https://user-images.githubusercontent.com/87286435/146700211-075a086d-573e-4596-8f5d-945244ca25aa.png)